app/components/GroupMember/index.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import React from 'react';
+import styles from './GroupMember.css';
+import { Link } from 'react-router';
+import cx from 'classnames';
+import type { User } from 'app/models';
+
+//Mangler å sortere medlemmer etter ansiennitet (dette skal ikke inn hit (blir vanskelig))
```

Comment:
Remove Norwegian comment and create an issue out of this that you can solve later 🥇